python_src/src/lamp_py/ingestion/config_busloc_trip.py

```diff
@@ -0,0 +1,89 @@
+from typing import List, Tuple
+import pyarrow
+
+from .gtfs_rt_detail import GTFSRTDetail
+from .gtfs_rt_structs import (
+    trip_descriptor,
+    vehicle_descriptor,
+    stop_time_event,
+)
+
+
+class RtBusTripDetail(GTFSRTDetail):
+    """
+    Detail for how to convert RT GTFS Trip Updates from json entries into
+    parquet tables.
+    """
+
+    def transform_for_write(self, table: pyarrow.table) -> pyarrow.table:
+        """modify table schema before write to parquet"""
+        return self.flatten_schema(
+            self.explode_table_column(
+                self.flatten_schema(table), "trip_update.stop_time_update"
+            )
+        )
```

Comment:
overloaded `transform_for_write` function does more than just flatten, also explodes the `trip_update.stop_time_update` field